content/12.privacy.md

```diff
@@ -1,31 +1,32 @@
+While deep learning continues to be a powerful, transformative tool within life sciences research—spanning basic biology and pre-clinical science to varied translational approaches and clinical studies—it is important to comment on ethical considerations.
+For instance, despite the fact that deep learning methods are helping to increase medical efficiency through improved diagnostic capability and risk assessment, certain biases may be inadvertently introduced into models related to patient age, race, and gender [@doi:10.1002/hast.977].
+Deep learning practitioners may make use of datasets not representative of diverse populations and patient characteristics [@doi:10.1377/hlthaff.2014.0048], thereby contributing to these problems.
 
-Therefore, it is important to think thoroughly and cautiously about deep learning applications and its potential impact to persons and society—mindful of possible harms, injuries, injustices, and other types of wrongdoings.
-At a minimum, practitioners must ensure that, wherever relevant, their life sciences projects are fully compliant with local research governance/approval policies, legal requirements, institutional review board (IRB) policies, and any other relevant bodies and their standards.
+Therefore, it is important to think thoroughly and cautiously about deep learning applications and their potential impact to persons and society—mindful of possible harms, injuries, injustices, and other types of wrongdoings.
+At a minimum, practitioners must ensure that, wherever relevant, their life sciences projects are fully compliant with local research governance and approval policies, legal requirements, institutional review board policies, and any other relevant bodies and standards.
 Moreover, we offer below three tangible, action-oriented recommendations to further empower and enrichen deep learning researchers.
 
-First, just as it is a best practive to keep a project-specific or programming-related issue tracker detailing known bugs and other technical issues, practitioners should get into the habit of keeping an active _ethics register_.
+First, just as it is a best practice to keep a project-specific or programming-related issue tracker detailing known bugs and other technical issues, practitioners should get into the habit of keeping an active _ethics register_.
 In this register, ethical concerns can be raised, recorded, and resolved, exactly as software problems are triaged and fixed. 
-Because projects using deep learning usually rely on writing code, an ethics register can be a part of the issue tracker in the version-control system for the software itself.
-By colocating the two, practioners can operationalize the concept that ethical problems arer "bugs" that must be resolved, not nice-to-haves that can be considered at some indefinite point in the future.
```

Comment:
I think the word we want here is "colocate" since the ethical concerns are located with coding concerns:

https://www.merriam-webster.com/dictionary/collocate
https://www.merriam-webster.com/dictionary/colocate

Reply:
Good catch.  I'll push a correction / reversion.